<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMassToFormula</class>
 <widget class="QWidget" name="FormMassToFormula">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>342</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelInputMass">
         <property name="text">
          <string>Input mass:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="LabeledText" name="textInputMass"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonGetFormula">
         <property name="toolTip">
          <string>Look for formulae that have the input mass with given criteria</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/return.png</normaloff>:/icons/return.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonShowImportMassFromFile">
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/view-more-horizontal.png</normaloff>:/icons/view-more-horizontal.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frameImportMassFromFile">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelImportMassFromFile">
          <property name="text">
           <string>Import from file:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="textMassImportFile">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonImportMassFromFile">
          <property name="toolTip">
           <string>Import a list of mass from file</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="labelAllowedElement">
         <property name="text">
          <string>Allowed elements:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textAllowedElement">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonAllowedElement">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip">
          <string>Select elements that are possible in the candidate formula</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
         </property>
         <property name="shortcut">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonShowMassModification">
         <property name="toolTip">
          <string>Set modification</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/electron.png</normaloff>:/icons/electron.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="FrameFormulaModification" name="frameMassModification"/>
     </item>
     <item>
      <widget class="QGroupBox" name="frameMassTolerance">
       <property name="title">
        <string>Tolerance</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="labelMassToleranceLeft">
            <property name="text">
             <string>Lower bound:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textMassToleranceLeft">
            <property name="toolTip">
             <string>The lower (left) limit of the allowed mass difference between input mass and mass of candidate formulae</string>
            </property>
            <property name="text">
             <string notr="true">-0.05</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelMassToleranceRight">
            <property name="text">
             <string>Upper bound:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textMassToleranceRight">
            <property name="toolTip">
             <string>The upper (right) limit of the allowed mass difference between input mass and mass of candidate formulae</string>
            </property>
            <property name="text">
             <string notr="true">0.05</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QRadioButton" name="radioMassToleranceAbsolute">
            <property name="text">
             <string>Absol&amp;ute(Da)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioMassToleranceRelative">
            <property name="text">
             <string>&amp;Relative(ppm)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="FrameFormulaList" name="frameResultFormula">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="frameStatus">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelStatus">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>254</red>
             <green>254</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>254</red>
             <green>254</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>254</red>
             <green>254</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFileLink">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>254</red>
             <green>254</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>252</red>
             <green>252</green>
             <blue>252</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>254</red>
             <green>254</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>254</red>
             <green>254</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>254</red>
             <green>254</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Open the &lt;a href=\&quot;#\&quot;&gt;result&lt;/a&gt;&amp;nbsp;&amp;nbsp;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameFormulaList</class>
   <extends>QFrame</extends>
   <header>widget/frameformulalist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FrameFormulaModification</class>
   <extends>QFrame</extends>
   <header>widget/frameformulamodification.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabeledText</class>
   <extends>QLineEdit</extends>
   <header>widget/labeledtext.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
